docs/installation.rst

```diff
@@ -19,17 +19,21 @@ On ``linux``, you can install the Python 3 version into ``$HOME/miniconda3`` wit
 
    $ wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
    $ bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda3
```

Comment:
Suggest using `$MINICONDA` environment variable (to refer to `$HOME/miniconda`) to simplify all the downstream examples since other users may have different preferences.